ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml

```diff
@@ -23,12 +23,24 @@ parameters:
 - name: DEPLOY_VERSION
   value: "v3.11"
 - name: TEST_IMAGE
-  value: "registry.svc.ci.openshift.org/azure/test-base:latest"
+  value: quay.io/kwoodson/bushslicer
```

Comment:
Revert this change.

Reply:
yeah, sorry, wasn't sure how to test this with a different value from the cli.